### PR TITLE
Re-introduce support for network status buffer topics

### PR DESCRIPTION
**Pull request by mamarley**
This includes adding support for automatically updating the
TopicViewBuffer text when the topic changes (with much help from
@justJanne)

### CODE REVIEW

File: QuasselDroid/src/main/java/com/iskrembilen/quasseldroid/Quasseldroid.java [hunk 1]
```diff
@@ -25,6 +27,7 @@ public void onCreate() {
         //Load current theme
         ThemeUtil.initTheme(this);
         BusProvider.getInstance().register(this);
+        context = this;
```
Comment (kenborge) [inline QuasselDroid/src/main/java/com/iskrembilen/quasseldroid/Quasseldroid.java]: Gussing these are changes that are left over from refactored code. It is never used

Comment (mamarley) [inline QuasselDroid/src/main/java/com/iskrembilen/quasseldroid/Quasseldroid.java]: It is used in Network for some strings in the status buffer topic formatting.

Comment (kenborge) [inline QuasselDroid/src/main/java/com/iskrembilen/quasseldroid/Quasseldroid.java]: oh yeah, doh i missed that it was public

Comment (kenborge) [inline QuasselDroid/src/main/java/com/iskrembilen/quasseldroid/Quasseldroid.java]: Though i think you could have just done Quasseldroid.getApplicationContext()

Comment (mamarley) [inline QuasselDroid/src/main/java/com/iskrembilen/quasseldroid/Quasseldroid.java]: Nope, there is no .getApplicationContext() method.

File: QuasselDroid/src/main/java/com/iskrembilen/quasseldroid/Network.java [hunk 2]
```diff
@@ -289,8 +293,10 @@ public int getCountUsers() {
     }
 
     private void updateTopic(){
-        if(statusBuffer!=null){
-            statusBuffer.setTopic("");
+        if(statusBuffer != null){
+            statusBuffer.setTopic(networkName + " (" + server + ") | "
+                    + Quasseldroid.context.getResources().getString(R.string.users) + ": "
+                    + userList.size() + " | " + Helper.formatLatency(latency, Quasseldroid.context.getResources()));
```
Comment (justjanne) [inline QuasselDroid/src/main/java/com/iskrembilen/quasseldroid/Network.java]: This creates double latency formatting, as the latency is also added by the updateSubtitle() method in MainActivity

Comment (mamarley) [inline QuasselDroid/src/main/java/com/iskrembilen/quasseldroid/Network.java]: The previous latency is client to core.  The new one is core to IRC.

Comment (justjanne) [inline QuasselDroid/src/main/java/com/iskrembilen/quasseldroid/Network.java]: hmm, okay. That makes sense then.

**Pull request merged**